### PR TITLE
Add tests for debug. Add method for get simulator logs. Fix some problems in tests. Change abi split test names.

### PR DESCRIPTION
**Pull request by vhristov5555**
…lems in tests. Change abi split test names.

### CODE REVIEW

File: core/utils/device/device.py [hunk 1]
```diff
@@ -31,6 +31,7 @@ def __init__(self, id, name, type, version):
             type = type.replace(',', '')
             type = type.replace('ProductType:', '').strip(' ')
             self.name = type
+            self.device_log = SimAuto.get_logs(self.id)
```
Comment (dtopuzov) [inline core/utils/device/device.py]: Can we rename it to make it clear it is path.

File: core/utils/debug.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        shadow_dom_element = self.driver.find_elements(By.CSS_SELECTOR, 'div.vbox.flex-auto.dimmed-pane')
+        if shadow_dom_element:
+            shadow_div_element = self.driver.execute_script(self.shadow_root_element_script, shadow_dom_element[0])
+            shadow_elements = self.get_shadow_element_in_shadow_dom(".vbox.flex-auto", shadow_div_element)
+            reconnect_button = self.get_element_by_css_selector_and_text("button", "Reconnect DevTools",
+                                                                         shadow_elements)
+            if reconnect_button is not None:
+                reconnect_button.click()
+
+    def wait_until_element_is_visible(self, by_expression, value, timeout=15):
```
Comment (dtopuzov) [inline core/utils/debug.py]: Can we move it to chrome since it is not debugger specific.

File: core/utils/device/device.py (hunk 1 above)
Comment (vhristov5555) [reply core/utils/device/device.py]: done

File: core/utils/debug.py (hunk 2 above)
Comment (vhristov5555) [reply core/utils/debug.py]: done

**Comment by vhristov5555** [comment]
test

**Pull request merged**